### ISSUE TITLE
👑 [需求]可以提供接口，控制ProLayout下的PageHeader组件渲染吗

### ISSUE THREAD

**Issue by kongdf123** [ant-design/pro-components#2280]
### 🥰 需求描述
直接上图
![image](https://user-images.githubusercontent.com/4360691/112092568-bcb05d80-8bd2-11eb-9da1-c8bc262ab368.png)

在功能区域里，面包屑下方，有时候不想展示这块，找了很久，也没找到API。已经查明，属于ProLayout下的PageContainer - PageHeader.

### 🚑 其他信息
已经截图，见前面。
如有解决方案，请不吝赐教，谢谢。

**Comment by mouthguo** [comment]
看了你说的 `PageContainer`，试试设置成 `false` ?

> https://github.com/ant-design/pro-components/blob/master/packages/layout/src/components/PageContainer/index.tsx#L161-L164

**Comment by kongdf123** [comment]
@mouthguo no, it doesn't work. I mean that there should be a prop provided by ProLayout component to control PageHeader inside of ProLayout, not PageContainer,
like below :
![image](https://user-images.githubusercontent.com/4360691/112251901-0b273000-8c97-11eb-9657-015da96a7148.png)

**Comment by mouthguo** [comment]
@kongdf123
i tried it like this : `<PageContainer title={false} >Hello World</PageContainer>` , and it works well. what did you do ?

And,
 if `ProLayout` provide `pageHeader={false}` to `hide PageHeader` as what you said ,
 `breadcrumb` should be still here ? or fly away ?

personally, i think you just NEED modify partial props in `PageContainer - PageHeader`,  so , focus what you DO NOT NEED maybe a better way;

------

我写了半天英语，写不动，惊觉你最开始用的中文啊大兄弟！
你最开始提的 `issue` 不是要 `PageHeader` 的 `title` 部分的渲染吗？
那么你后面提的 `pageHeader={false}` 不就直接把整个部分，包括面包屑也都隐藏掉了吗？

**Comment by kongdf123** [comment]
@mouthguo 首先非常感谢回复，^_^, 是这样的，我想修改BasicLayout组件下的部分内容（PageLyout -> PageContainer -> PageHeader），
通过看源码了解到，ProLayout封装了PageContainer，那既然包含PageContainer,就想着是不是提供一些入口，可以微调PageContainer组件,现在问题是ProLayout没有开放一个属性或方法，无法access 到内部封装的PageContainer。
希望我能一次把问题说明白，感谢兄弟的耐心。

**Comment by mouthguo** [comment]
客气啦～
似乎一般都是
```javascript
<ProLayout>
    <PageContainer />
</ProLayout>
```
的形式来组织页面的，所以感觉需要控制这个的话，直接在 `PageContainer` 上设置 `title={false}` 就可以了。

我理解，你想要的，是类似 `Form` 提供了 `labelCol` ，可以协助 `Form.Item` 做 common 布局， `Form.Item` 依然可以修改 `labelCol` 来微调这种。

不过看代码，`BasicLayout` 里面那个 `context` 真的包含了很多东西了已经
